### DISCUSSION TITLE
even darker theme?

### DISCUSSION THREAD

**Discussion by ghost** [dracula/dracula-theme#610] [Ideas] [answered]
I think the background is a little bit too light for me, it would be nice to have a "distribution" of it which has the background a little darker. Kind of like dracula-at-night but not only for VS Code.

(transferred from [here](https://github.com/dracula/dracula-theme/issues/607))

**Comment by bigpick** [comment]
Hey! Thanks for moving this over to a discussion.

As for your ask, have you considered looking at Dracula Pro? I think particularly **Van Helsing** sub-theme would be of interest to you:

### [Dracula Pro Van Helsing subtheme](https://draculatheme.com/pro)
![](https://draculatheme.com/static/img/pro/vscode/6.png)

... as a bonus, comparing that to your reference dracula-at-night alternative:

![](https://dzwonsemrish7.cloudfront.net/items/2q3r15432s3H00013u1j/Screen%20Shot%202018-06-02%20at%2010.16.01.png)

it's even darker too!

**Comment by ghost** [comment]
for me dracula pro kinda looks ugly, and i dont really like it being paid tbh lol

**Reply by bigpick** [reply]
Van Helsing is the official _even darker_ option imo, so if you don't like that or any of the other ones there, or don't want to support them, I think your best option would be forking and modifying/maintaining your own offshoot to your liking. 

Given the additional variety Dracula Pro has in combination with the amount of other non-Dracula work the main owners are doing, I very strongly doubt another official offshoot would be warranted, but that's just my take on it

**Reply by ghost** [reply]
yeah, im gonna make my own dracula clone

**Comment by ghost** [comment, marked as answer]
I made my own distribution of Dracula called "Darkerula". This answers this discussion.
